package.json

```diff
@@ -15,6 +16,12 @@
     "node",
     "daemon"
   ],
+  "standard": {
+    "ignore": [
+      "/test/*",
```

Comment:
done in https://github.com/ipfs/node-ipfsd-ctl/commit/c032796f90474d99ffa9650dc2ec8dda1e3a2168